lib/ex_doc/cli.ex

```diff
@@ -53,11 +66,19 @@ defmodule ExDoc.CLI do
   end
 
   defp parse_args([_project, _version, _source_beam] = args), do: args
+
   defp parse_args([_, _, _ | _]) do
     IO.puts "Too many arguments.\n"
     print_usage()
     exit {:shutdown, 1}
   end
+
+  defp parse_args(:version) do
```

Comment:
nah.. i was testing with the file when I first grabbed it. I left it accidentally. sorry about that. 
pushed!